### PR TITLE
Added some context to error messages.

### PR DESCRIPTION
**Pull request by fredrik-jansson-se**
Added some context to error messages that bit me when trying this out.

### CODE REVIEW

**Review by ionut-arm — APPROVED** [review]
Hi 👋🏻 Thanks for the changes, this is an area that we've identified/been made aware of for UX enhancements. Hopefully it wasn't too much of a headscratch to get over them!

**Review by anta5010 — APPROVED** [review]
LGTM.
Thank you for the PR

File: src/utils/config.rs
Comment (fredrik-jansson-se) [reply src/utils/config.rs:282]: I'll fix that!

**Comment by fredrik-jansson-se** [comment]
Hey, thank you!

Thanks for make all this available, from my very limited testing, it looks like this could really help us!

**Comment by ionut-arm** [comment]
> Thanks for make all this available, from my very limited testing, it looks like this could really help us!

More than happy to help. Any chance you can give an overview of your use case? (asking mainly out of curiosity 😄 )

Also, I think your commits are missing the [DCO signoff](https://github.com/parallaxsecond/parsec/pull/618/checks) - we'll need those before merging.

**Comment by fredrik-jansson-se** [comment]
Ok, I "signed off" the last commit, is that alright?

Investigating if we (https://avassa.io) could use this as our abstraction for HSM/TPM for the cases there are e.g. TPM modules available in the hardware. 

Our general use case is to enable our customers to distribute applications (containers) to their edge locations, e.g. retail stores, buildings, factories etc.

**Comment by ionut-arm** [comment]
> Ok, commits are now signed off.

Excellent, thank you!

> Investigating if we (https://avassa.io) could use this as our abstraction for HSM/TPM for the cases there are e.g. TPM modules available in the hardware.
> 
> Our general use case is to enable our customers to distribute applications (containers) to their edge locations, e.g. retail stores, buildings, factories etc.

Nice, that's pretty much the space Parsec was made for. If you'd like to give any feedback, get some help with anything Parsec-related, or just to be involved in general, we're quite active on our [public Slack channel and community meeting](https://github.com/parallaxsecond/community).

**Review by MattDavis00 — APPROVED** [review]
LGTM

**Pull request merged**
